tree/ntuple/v7/inc/ROOT/RNTupleOptions.hxx

```diff
@@ -46,14 +46,21 @@ All page sink classes need to support the common options.
 class RNTupleWriteOptions {
   int fCompression{RCompressionSetting::EDefaults::kUseAnalysis};
   ENTupleContainerFormat fContainerFormat{ENTupleContainerFormat::kTFile};
+  // Whether the user has explicity set these options, e.g. to override a TFile's compression setting.
+  bool fCompressionOverride = false;
```

Comment:
--> `fIsCompressionOverride`

Reply:
Thank you, fixed in the rebased b3c9227e1f6aeeb0baa412f2c99f1677059bcf3c.